examples/predicates/src/lib.rs

```diff
@@ -89,10 +70,32 @@ mod tests {
         let signatures = [signature1, signature2, signature3];
         // ANCHOR_END: predicate_signatures
 
+        // ANCHOR: predicate_load
+        abigen!(Predicate(name="MyPredicate", abi="packages/fuels/tests/predicates/predicate_signatures/out/debug/predicate_signatures-abi.json"));
+
+        let mut predicate: Predicate = MyPredicate::load_from(
+            "../../packages/fuels/tests/predicates/predicate_signatures/out/debug/predicate_signatures.bin",
+        )?.set_data(signatures).get_predicate();
```

Comment:
> Mainly because the MyPredicate ... .get_predicate() looks a bit weird, getting a predicate from a predicate.

I thought the same but didn't come up with a nice solution like you, @segfault-magnet. This makes a lot of sense to me, and it's worth delaying this a bit more to get to a more friendly experience. 

If we're all in agreement (including you @Salka1988 ), I'd say let's do it!